spec/integration/driver_spec.rb

```diff
@@ -56,6 +56,29 @@ def session_url(path)
       end
     end
 
+    unless Capybara::Poltergeist.jruby?
```

Comment:
instead of of not running the test when jruby, please set the test to pending when jruby is true so that we can see the results in the logs as an accepted failure